contracts/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "gasp-contracts",
+  "version": "1.0.0",
+  "description": "Gasp smart contracts for an EVM compatible virtual machine",
+  "scripts": {
+    "format": "bun prettier --check '**/*.{json,md,yml}' && forge fmt --check",
+    "format:fix": "bun prettier --write '**/*.{json,md,yml}' && forge fmt",
+    "lint": "bun solhint '{script,src,test}/**/*.sol'",
+    "lint:fix": "bun solhint --fix '{script,src,test}/**/*.sol'",
+    "analyze": "slitherin . --separated",
+    "analyze:ci": "slitherin --separated . --skip-clean --ignore-compile"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/gasp-xyz/gasp-monorepo/contracts"
+  },
+  "license": "BUSL-1.1",
+  "private": true,
+  "maintainers": [
+    "mateuszaaa <mateusz.cz.nowakowski@gmail.com>",
+    "Serge <2901744+evercoinx@users.noreply.github.com>"
+  ],
+  "devDependencies": {
+    "prettier": "^3.3.3",
+    "solhint": "^5.0.3"
+  }
+}
```

Comment:
we have bunch of smaller services already in the repository, each of them defining its own toolset, ideally we should have some single config, but until we get there i would suggest putting configs into directory that utilizes it :thinking: